cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -833,32 +829,6 @@ public Tracing getTracing() {
         return tracing;
     }
 
-    /**
-     * Creates the ClusterRoleBinding which is used to bind the Kafka Connect SA to the ClusterRole
-     * which permissions the Kafka init container to access K8S nodes (necessary for rack-awareness).
-     *
-     * @return The cluster role binding.
-     */
-    public ClusterRoleBinding generateClusterRoleBinding() {
```

Comment:
Since KMM2 inherits from KafkaConnect, why does this need to be removed / moved?

Reply:
OK, thanks.